packages/google_maps_flutter/google_maps_flutter_ios/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.4.2
+
+* Fixes a bug in takeSnapShot function that incorrectly returns a blank image on iOS 17
```

Comment:
The function is called `takeSnapshot`, not `takeSnapShot` (and should also be enclosed in backticks, since it's a code reference.

Reply:
done.